Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>*.iml\r\n.gradle\r\n/local.properties\r\n/.idea/caches\r\n/.idea/libraries\r\n/.idea/modules.xml\r\n/.idea/workspace.xml\r\n/.idea/navEditor.xml\r\n/.idea/assetWizardSettings.xml\r\n.DS_Store\r\n/build\r\n/captures\r\n.externalNativeBuild\r\n.cxx\r\nlocal.properties\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 9ec9920fd42ce10d2b1dc54ee3f276ac651bb664)
+++ b/.gitignore	(date 1736244336586)
@@ -13,3 +13,12 @@
 .externalNativeBuild
 .cxx
 local.properties
+/.idea/.name
+/.idea/compiler.xml
+/.idea/gradle.xml
+/.idea/misc.xml
+/.idea/runConfigurations.xml
+/.idea/shelf/Uncommitted_changes_before_Update_at_07_01_2025_14_02_[Changes]/shelved.patch
+/.idea/shelf/Uncommitted_changes_before_Update_at_07_01_2025_14_03_[Changes]/shelved.patch
+/.idea/vcs.xml
+/.idea/shelf/Uncommitted_changes_before_Update_at_07_01_2025_14_02_[Changes]/shelved.patch
